src/dcurl.c

```diff
@@ -48,6 +51,11 @@ extern ImplContext PoWCL_Context;
 extern ImplContext PoWFPGAAccel_Context;
 #endif
 
+#if defined(ENABLE_REMOTE)
+extern RemoteInterfaceContext RemoteInterface_Context;
+static uv_sem_t notify_remoteinterface;
```

Comment:
It is hard to read the variable name.
I think it should be simplified to `notify_remote` or at least it should be `notify_remoteInterface`.